utils/dragster.ts

```diff
@@ -9,20 +9,22 @@ type Options = {
 }
 
 export default function dragster(query: string, options: Options) {
-    const noop = () => {}; // eslint-disable-line no-empty-function
+    const preventDefault = (e: Event) => {
```

Comment:
Are these changes to these default handlers needed when `FileUpload` overrides all of these anyway?

If I'm reading this correctly, it means that all unhandled drag events will be blocked as soon as someone else starts using dragster

Reply:
@hmhealey 
Thank you for your review!

> If I'm reading this correctly, it means that all unhandled drag events will be blocked as soon as someone else starts using dragster

Yes, you are correct.
But I didn't change behavior of dragster.
dragenter, dragleave, dragover and drop called preventDefault before I made this changes.
But I needed to lifted preventDefault up to FileUpload so I replaced noop with preventDefault to keep behavior compatible.